content/docs/user-guide/contributing/core.md

```diff
@@ -56,13 +56,13 @@ do that, we **strongly** recommend creating a
 $ cd dvc
 $ python3 -m venv .env
 $ source .env/bin/activate
-$ pip install -e ".[all,tests]"
+$ python3 -m pip install -e ".[all,tests]"
```

Comment:
☝🏼 

For context I personally had to do this on my Mac since `pip` was still from globals even after activating the venv.

Reply:
p.s. could be just `python` at this point but I thought `python3` looked more consistent in that sample block (same result).

Reply:
This shouldn't be necessary, once the venv is activated the `pip` in the user's `$PATH` should be the one from the venv's `bin/pip`.

In your case it sounds like a particular issue w/your shell environment configuration (you probably needed to run zsh's `rehash` to force zsh to handle `$PATH` changes).

Reply:
Hmmm ah yes. Zsh seems to alias `pip` to `noglob pip` by default...

Reply:
p.s. but under the alias the `pip` used from $PATH is correct 🎉  so yeah just a confusion on my side.

```console
$ which -a pip
pip: aliased to noglob pip
/Users/ME/dvc/.env/bin/pip
/opt/homebrew/bin/pip
```